demos/common/cpp/utils/include/utils/common.hpp

```diff
@@ -249,3 +250,69 @@ inline void logExecNetworkInfo(const InferenceEngine::ExecutableNetwork& execNet
         }
     }
 }
+
+inline
+void logCompiledModelInfo(
+    const ov::runtime::CompiledModel& compiledModel,
+    const std::string& modelName,
+    const std::string& deviceName,
+    const std::string& modelType = "")
+{
+    slog::info << "The " << modelType << (modelType.empty() ? "" : " ") << "model " << modelName << " is loaded to " << deviceName << slog::endl;
+    std::set<std::string> devices;
+    for (const std::string& device : parseDevices(deviceName)) {
+        devices.insert(device);
+    }
+
+    if (devices.find("AUTO") == devices.end()) { // do not print info for AUTO device
+        for (const auto& device : devices) {
+            try {
+                slog::info << "\tDevice: " << device << slog::endl;
+                std::string nstreams = compiledModel.get_config(device + "_THROUGHPUT_STREAMS").as<std::string>();
+                slog::info << "\t\tNumber of streams: " << nstreams << slog::endl;
+                if (device == "CPU") {
+                    std::string nthreads = compiledModel.get_config("CPU_THREADS_NUM").as<std::string>();
+                    slog::info << "\t\tNumber of threads: " << (nthreads == "0" ? "AUTO" : nthreads) << slog::endl;
+                }
+            }
+            catch (const ov::Exception&) {}
+        }
+    }
+}
+
+inline
+void logBasicModelInfo(const std::shared_ptr<ov::Model>& model) {
+    slog::info << "model name: " << model->get_friendly_name() << slog::endl;
+
+    // Prepare input blobs
```

Comment:
You don't modify them thus it's not preparation

Reply:
yes, corrected